src/modules/auth/auth.service.ts

```
@@ -423,4 +460,37 @@ export class AuthService {
+
+    // 1️⃣ Get admin token
+    const adminTokenData = await this.keycloakService.getAdminKeycloakToken();
+    const adminToken = adminTokenData?.access_token;
+
+    // 2️⃣ Find user in Keycloak
+    const user = await this.keycloakService.getUserByUsername(username);
+    if (!user) {
+      throw new HttpException('USER_NOT_FOUND', HttpStatus.NOT_FOUND);
+    }
+
+    // 3️⃣ Call reset password API
```

ESLint

[error] Replace `····` with `↹↹`

(prettier/prettier)